Makefile

```diff
@@ -189,6 +189,7 @@ sync-docs:
 ###############################################################################
 
 build-docker:
+	make build-linux
```

Comment:
This should be a dependency, not called explicitly - i.e. `build-docker: build-linux`.

Reply:
ah yes will make the change.